Fabric.Identity.API/Models/CustomGrantType.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Fabric.Identity.API.Models
+{
+    public static class CustomGrantType
```

Comment:
Is this an API model, or is it more of a constant? If its a constant, should we move it to the constants namespace?

Reply:
Good point - IS4 put their GrantType.cs in a Models namespace, but I agree our Models namespaces has a different meaning. I'll move it.